homeassistant/components/srp_energy/sensor.py

```diff
@@ -0,0 +1,153 @@
+"""Support for SRP Energy Sensor."""
+from datetime import datetime, timedelta
+import logging
+
+import async_timeout
+from requests.exceptions import ConnectionError as ConnectError, HTTPError, Timeout
+
+from homeassistant.const import ATTR_ATTRIBUTION, ENERGY_KILO_WATT_HOUR
+from homeassistant.helpers import entity
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import (
+    ATTRIBUTION,
+    DEFAULT_NAME,
+    DOMAIN,
```

Comment:
```suggestion
    DOMAIN as SRP_ENERGY_DOMAIN,
```
DOMAIN in platforms are reserved for the platform itself, in this case sensor